packages/components/psammead-leading-story-promo/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Leading Story Promo Changelog
+
+| Version | Description |
+|---------|-------------|
+| 1.0.0-alpha.0| [PR#2399](https://github.com/bbc/psammead/pull/2399) Initial creation of package. |
```

Comment:
From this https://github.com/bbc/psammead/pull/2428, we will be setting the first version to `0.1.0-alpha.1`.